js/popup.js

```diff
@@ -0,0 +1,71 @@
+const overlay = document.querySelector('#overlay');
+const openModalButton = document.querySelectorAll('[data-modal-target]');
+
+const mobileModalContent = [{
+  title: 'Multi-post stories',
+  closeWork: '<i class="closeWork fa fa-times"></i>',
+  displayImgSmall: './images/modal-mobile.png',
+  descriptionMobile: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+  diplayUl: ['|', 'html', '|', 'Ruby on rails', '|', 'css'],
+}];
+
+const desktopModalContent = [{
+  title: 'Multi-post stories',
+  closeWork: '<i class="closeWork fa fa-times"></i>',
+  displayImgLarge: './images/Snapshoot-Portfolio-lg.png',
+  descriptionDesktop: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+  diplayUl: ['|', 'html', '|', 'Ruby on rails', '|', 'css', '|', 'Github'],
+}];
```

Comment:
- [ ] You are making an array of objects but you should not make different objects for mobile and desktop. Please make only one object for both orientations :+1:
- [ ] Add all the projects here not just one :+1:

Reply:
- Thanks for your feedback I can create one array for both but the value or pair for the list and image is different so how can I fix that? popup.js(line 7 and 15, lines 9 and 17) thanks so much.